_config.yml

```diff
@@ -1,9 +1,9 @@
-# TODO: replace the title with the name of your restaurant!
+: replace the title with the name of your restaurant!
```

Comment:
your site isn't building, because the `#` sign to cerate a comment got wipes out. If you wanted the site back, you could add it in again